src/Command/CompileCommand.php

```diff
@@ -34,6 +34,10 @@ protected function buildOptionParser(ConsoleOptionParser $parser): ConsoleOption
             'help' => 'The file or plugin you want to compile.',
         ]);
 
+        $parser->addOption('view', [
```

Comment:
A more speaking name `view-class` would be preferable.

Reply:
Ok, done.